### PR TITLE
drop debian 8 support

### PR DESCRIPTION
**Pull request by bastelfreak**
Acceptance tests proved that debian 8 wasn't correctly working. Is it still a major release if we officially drop it? I guess so.

### CODE REVIEW

**Comment by bastelfreak** [comment]
for reasons I don't know github didn't display the commit message in the PR box thigny:

the upstream packages for debian 8 are broken, they miss the init files. Details are in #139. Also it is EOL and we support Debian 9. We were unable to get the acceptance tests working.

**Comment by dhollinger** [comment]
I think this is an issue more of the application than of the OS. 

If Gluster doesn't properly support Debian 8, then I don't think we can, in good conscience, support Debian 8 either. I vote that we open an issue/ticket with the maintainer of the Debian 8 Gluster package(s) and reference that issue/ticket in the commit to drop Debian 8 support. That way, if the issue were to get fixed while Debian 8 is still supported by the community, we could then re-enable support.

TL;DR - We should drop Debian 8 support until Gluster is fixed there.

**Comment by wyardley** [comment]
As @bastelfreak mentioned elsewhere, I think that 3.12.2-2 is the correct version, and the one that's linked from their download page.

https://download.gluster.org/pub/gluster/glusterfs/LATEST/Debian/8/apt/dists/jessie/main/binary-amd64/Packages

With that version, there are apparently some SELinux problems, that's something we deal with in other modules, so we should just deal with that in the usual way (or disable it in the acceptance test and make a note of it in the README, since plenty of people don't use it).

**Comment by bastelfreak** [comment]
As suggested I will close this and create a new PR to bump the default gluster version
